qemu/qemu_watcher.py

```diff
@@ -493,7 +493,9 @@ def on_qbus_realize(self):
     def on_bus_unparent(self):
         # bus_unparent, before actual unparenting
 
-        """ hw/core/bus.c:123 v2.12.0
+        """
+            hw/core/bus.c:118 v4.1.0
```

Comment:
Why was't it inserted in the end of the positions list?

Reply:
Because it is likely that newer version of Qemu is used.

Reply:
There is no requirement to sort by new version.

Reply:
There is no a contrary requirement too.